src/ome_zarr_models/v04/image.py

```diff
@@ -71,6 +72,11 @@ class ImageAttrs(Base):
         min_length=1,
     )
     omero: Omero | None = None
+    image_labels: Annotated[ImageLabel | None, Field(..., alias="image-label")] = None
```

Comment:
are we allowed by the spec to write this to JSON as `null` if the field is unset?

Reply:
Probably not. We can use the `exclude_unset` keyword argument to `model_dump()` to make sure we're not dumping any unset, but I think that still leaves open the option that someone sets the value to `None` then writes 😢 looking at https://docs.pydantic.dev/latest/migration/#required-optional-and-nullable-fields I'm not sure there's a way round this though - perhaps we should just document it as a known issue?

Reply:
over in `pydantic-ome-ngff` I used a special attribute to track the fields that shouldn't be serialized if they are null. we might want to do something similar. But until then I would suggest we keep this as-is and put in a TODO statement about dealing with the null value

Reply:
👍 I've put in a "known issue" note on the homepage of the docs, definiltey up for us fixing this in a follow up though